edx-platform/learninghub-theme/lms/templates/instructor/edx_ace/enrollenrolled/email/body.html

```diff
@@ -1,95 +1,184 @@
+                                {{ site_name }}
+                            </a><wbr>&nbsp;
+
+                            <span style="color:#333333">
+                                {% if full_name%}
+                                    to {{ full_name }}
+                                {% else %}
+                                    to {{ email_address }}
+                                {% endif %}
+                            </span>
+                        </span>
+                    </span>
+                </td>
+            </tr>
+
+            <tr>
+                <td align="center" style="
+                    color: white;
+                    font-size: 12px;
+                    font-family: Arial,Helvetica,sans-serif;">
+                    <div style="
+                        text-align: center;
+                        background: #007FA3;
+                        padding: 15px 0 15px 0;">
+                        Pearson Education Ltd 2018 &nbsp; &nbsp; Registered No: 872828 &nbsp; &nbsp;
+                        <a href="https://www.google.com/maps/place/80+Strand,+London+WC2R+0RL,+UK" target="_blank" data-saferedirecturl="https://www.google.com/url?q=https://www.google.com/maps/place/80%2BStrand,%2BLondon%2BWC2R%2B0RL,%2BUK&amp;source=gmail&amp;ust=1571217499022000&amp;usg=AFQjCNF60-Jsl55ZzfPThyfsZj4phbX1sw" style="
+                            color: white;
+                            text-decoration: underline;
+                            font-family: Arial,Helvetica,sans-serif;font-size:12px">
+                            80 Strand London WC2R 0RL
+                        </a>
+                        <br>
+                    </div>
+
+                    <div style="
+                        text-align: center;
+                        background: #005A70;
+                        padding: 15px 0 15px 0;">
+                        <a href="https://uk.pearson.com/pearson-privacy-and-you/privacy-policy.html" style="color:white;text-decoration:underline;font-family:Arial,Helvetica,sans-serif;font-size:12px" target="_blank" data-saferedirecturl="https://www.google.com/url?q=https://uk.pearson.com/pearson-privacy-and-you/privacy-policy.html&amp;source=gmail&amp;ust=1571217499022000&amp;usg=AFQjCNGdyavdYzMHZFc65fm1B5OYdqc6uQ">
+                        Privacy policy
+                        </a>&nbsp; &nbsp;
+
+                        <span style="font-family:Arial,Helvetica,sans-serif;font-size:12px">
+                            &nbsp;
+                        </span>
+
+                        <a href="http://uk.pearson.com/pearson-privacy/your-data-at-pearson/cookies/cookie-policy" target="_blank" data-saferedirecturl="https://www.google.com/url?q=http://uk.pearson.com/pearson-privacy/your-data-at-pearson/cookies/cookie-policy&amp;source=gmail&amp;ust=1571217499022000&amp;usg=AFQjCNGslgDqTVRgk9s8KkOpSF5v336jIA" style="color:white;font-family:Arial,Helvetica,sans-serif;font-size:12px">
```

Comment:
Please, split the attributes on their own line to avoid line too long. @andrey-canon
Please, add a space after the colon. @andrey-canon